interpolate.go

```diff
@@ -521,6 +528,10 @@ func encodeValue(buf []byte, arg interface{}, flavor Flavor) ([]byte, error) {
 			case SQLServer:
 				buf = append(buf, "0x"...)
 				buf = appendHex(buf, data)
+
+			case ClickHouse:
+				buf = append(buf, "_binary"...)
```

Comment:
I don't find `_binary` method in ClickHouse doc. Instead, I find [`unhex`](https://clickhouse.com/docs/en/sql-reference/functions/encoding-functions#unhex) seems the most appropriate method to convert arbitrary data to blob. You can try to write logic against `unhex`, or you can provide me a pointer to `_binary` if this method works well in ClickHouse.